ci-operator/step-registry/telco5g/cnf-tests/telco5g-cnf-tests-commands.sh

```diff
@@ -110,6 +110,9 @@ function deploy_and_test {
         sed -i "s/name\: performance/name\: performance-sriov/g" perf_profile_for_sriov.yaml
         sed -i "s/worker-cnf/${node_label}/g" perf_profile_for_sriov.yaml
         oc apply -f perf_profile_for_sriov.yaml
+
+        # Deploy multinetwork policy feature to run the suite
+        FEATURES_ENVIRONMENT="${features_env}" FEATURES="multinetworkpolicy" make feature-deploy
```

Comment:
there is this one https://github.com/openshift/release/pull/33349/files#diff-95f348f9df3a27a76247532a27f89145745af81dd5efe90919aa5e41c38ccaa5R129

so maybe you can just add the `multinetworkpolicy` to the `feature` variable

Reply:
Can this feature be tested in parallel with other sriov tests?
The way this script works is by taking one feature from FEATURES list, deploy and test it in a thread. So all features are deployed and tested in parallel.